hibernate/src/org/netbeans/modules/hibernate/resources/HibernateRevengTemplate.xml

```diff
@@ -1,2 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
From the name this looks to be a template for a user file. If so, I would not include an apache license here, if it ends up int he users code base. There is no real IP here, so I'd be just annoyed if I'd have to remove/replace the license by hand.

Reply:
Okay, I see.